packages/turbo/ios/RNSessionManager.swift

```diff
@@ -0,0 +1,27 @@
+//
+//  RNSessionManager.swift
+//  RNTurbo
+//
+//  Created by Patryk Klatka on 22/11/2023.
+//
+
+import Foundation
+
+class RNSessionManager {
+
+  private var sessions: [NSString: RNSession] = [:]
+  static var shared: RNSessionManager = RNSessionManager()
+
+  private init() {}
+
+  func findOrCreateSession(sessionHandle: NSString?, webViewConfiguration: WKWebViewConfiguration) -> RNSession {
+    let resolvedSessionHandle = sessionHandle ?? ("Default" as NSString)
```

Comment:
I think we should make `sessionHandle` required and remove this. Why would we call `findOrCreateSession(..)` without an explicit handle?

JS Component could provide it's default handle, so having stricter requirements on native side I think is better.

Reply:
> JS Component could provide it's default handle, so having stricter requirements on native side I think is better.
It actually does, will remove it.